From adb4c02c92ba33b73ff26d5bc39fed82388d8824 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Thu, 28 Oct 2021 01:02:03 +0300
Subject: [PATCH 47/89] defconfig: Fix mipi-csi camera

Regression: the driver for mipi-csi camera ov5640 has been dropped since
the configuration parameter CONFIG_MEDIA_SUPPORT_FILTER has been enabled by
default in Kconfig. This parameter advertently intended to guard us from
configuring the subsystem we hardly comprehend.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/configs/cl-imx8m-mini_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/cl-imx8m-mini_defconfig b/arch/arm64/configs/cl-imx8m-mini_defconfig
index a764b4022657..c4f1603773e8 100644
--- a/arch/arm64/configs/cl-imx8m-mini_defconfig
+++ b/arch/arm64/configs/cl-imx8m-mini_defconfig
@@ -814,3 +814,4 @@ CONFIG_CORESIGHT=y
 CONFIG_CORESIGHT_LINK_AND_SINK_TMC=y
 CONFIG_CORESIGHT_SOURCE_ETM4X=y
 CONFIG_RESET_IMX7=y
+CONFIG_MEDIA_SUPPORT_FILTER=n
-- 
2.17.1

